OracleWebLogic/samples/12213-domain-wdt/build-archive.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+#Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+#
+#Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+rm -Rf archive
```

Comment:
This is potentially an exceptionally dangerous command, depending on where the script is run. I think you should test for the existence of the `archive` directory first and be more specific about what you're deleting.